bootstrap/gwcheck.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+# -- GATEWAY CHECK MODE --
+# This mode merely responds with information about the request. This can be
+# used by an operator or automated test suite to ensure that the channel
+# through which requests reach the application code (web server, gateway
+# interface) is properly set up
+
+# * Mode enabling flag
+# Check for flags that enable the operation of this mode
+#  file: gwcheck.enabled , in the same folder along this file
+#  environment: USH_PLATFORM_GWCHECK_ENABLED variable
+#    (NOTE that the .env file in the base folder is NOT parsed for this script!)
+$enabled =
+    file_exists(__DIR__ . '/gwcheck.enabled') ||
+    ($_ENV['USH_PLATFORM_GWCHECK_ENABLED'] ?? null);
+if (!$enabled) {
+    # While disabled, we indicate that in a special header
+    header("X-Ushahidi-Platform-GWCheck: off");
+    http_response_code(204);
+    exit();   # -- END request processing
+} else {
+    header("X-Ushahidi-Platform-GWCheck: on");
+}
+
+# make the origin header handy
+$origin = $_SERVER['HTTP_ORIGIN'] ?? null;
+
+# * CORS pre-flight request mode check
+if ($_SERVER['REQUEST_METHOD'] == 'OPTIONS') {
+    # Check required headers
+    $acr_method = $_SERVER['HTTP_ACCESS_CONTROL_REQUEST_METHOD'] ?? null;
+    $acr_headers = $_SERVER['HTTP_ACCESS_CONTOL_REQUEST_HEADERS'] ?? null;
```

Comment:
small typo here, should be CONTROL (says "CONTOL", took me a while to see it sorry😬  )
With this fix, works great 🎉

Reply:
:D thanks for catching this. Fixing and merging :+1: